### DISCUSSION TITLE
Keypad Bezel print

### DISCUSSION THREAD

**Discussion by liamamiller** [CircuitSetup/Time-Circuits-Display#32] [Q&A]
Hi all. Any tips on printing the keypad bezel with Cura? I've tried twice with support enabled and it's snapped both times as I'm trying to remove the support structure. Taking a break from it for a day or two now. What settings did you use to print yours? I don't see any documentation on settings for printing certain/any parts.

Cheers!

**Comment by Ricky3Nut** [comment]
It's been awhile since I printed mine so I'd have to go back and see if I can recall anything from looking at my print file. But with Cura I typically set my support density at 10% or 15% maximum so its easier to remove. 

I vaguely recall it being a little tedious to cleanup and I used my cutting snips to trim out the underside just tried to take time and care.

Sorry, I know that's not very much help!

⁣Get BlueMail for Android ​

**Comment by liamamiller** [comment]
@Ricky3Nut - appreciate the advice! I'll have another go at it in a few days and hopefully will have some better luck.

**Comment by CircuitSetup** [comment]
This file is actually for Cura: https://github.com/CircuitSetup/Time-Circuits-Display/blob/master/Hardware/3d%20Files/cura_keypad_bezel-v2.3mf

You should be able to use the support settings from that. Also, ironing is turned on so the top surface is smooth.
